src/greyVest/Style.js

```diff
@@ -256,6 +256,7 @@ export default () => (
       .gv-text-button {
         border-radius: 100px;
         padding: 5px;
+        margin: -4px 0 -6px;
```

Comment:
We should document why we're doing this. It's a reasonable solution, but not the reasoning isn't obvious from this line

Reply:
This feel wrong. Why is this button specifying its margins?

Reply:
Added comment

Reply:
@stellarhoof this meant to be inline text/icon button. It specifies extra padding to have a larger hover area. But if padding is added to the vertical height this button breaks out of vertical line-height. Here is this button on hover, so you can see how much vertical space it takes with that padding:
<img width="165" alt="Screen Shot 2020-08-31 at 1 16 26 PM" src="https://user-images.githubusercontent.com/470292/91747886-02e99500-eb8d-11ea-9c05-a5327b4b1484.png">

Without this negative margin, it makes the whole section header jump in height when you open the section

Reply:
In that case we can set a min height on that filter header or toggle the button's visibility or something else. I don't think top-level dom nodes should specify their margins since that's a parent layout concern. If we make this change, every text button will define its outer spacing.

Reply:
Alright, that makes sense. I felt like fixing it right next to the cause and make text buttons fit into any text line without bloating it. But your point about concern separation is also valid. I will remove this and change the button's visibility and inner padding in the filter header.